### DISCUSSION TITLE
Man page vs Wiki

### DISCUSSION THREAD

**Discussion by kcthrn** [leo-arch/clifm#147] [Q&A] [answered]
If I consult the man page, am I missing out on anything in the wiki?

**Comment by leo-arch** [comment, marked as answer]
Hi @kcthrn. I try to keep both in synchro, but there are a few things more elaborated in the Wiki (like the file details section for long view mode, which, however, is documented in the main configuration file). As a rule of thumb, the manpage is enough, but, if you feel something needs more explanation, you can consult the Wiki (or just open a new issue asking for improved documentation).

**Reply by kcthrn** [reply]
Thanks!
